src/containers/AssetList.js

```diff
@@ -1,58 +1,110 @@
-import React from 'react';
+import React, {Component} from 'react';
 import { Link } from 'react-router-dom'
```

Comment:
In testing I get the following:

```
./src/containers/AssetList.js
  Line 1:  'Component' is defined but never used  no-unused-vars
  Line 2:  'Link' is defined but never used       no-unused-vars
```

It's so trivial that I'll just update the PR to remove these and then accept.